src/main/resources/assets/blocky-bubbles/lang/ru_ru.json

```
@@ -0,0 +1,13 @@
+{
+  "blocky-bubbles.options.culling_awareness.enum.bubble_column": "Колонна пузырей (подобная Java)",
+  "blocky-bubbles.options.culling_awareness.enum.non_air": "Невоздушные (паритет)",
+  "blocky-bubbles.options.culling_awareness.enum.none": "Без отбраковки",
+  "blocky-bubbles.options.culling_awareness.name": "Режим отбраковки",
+  "blocky-bubbles.options.culling_awareness.tooltip": "Какие блоки должны вызывать отбраковку соседних граней колонны пузырей.",
+  "blocky-bubbles.options.enable_animations.name": "Включить анимацию",
+  "blocky-bubbles.options.enable_animations.tooltip": "Следует ли анимировать текстуры колонны пузырей (требует перезагрузки ресурсов).",
+  "blocky-bubbles.options.quality.name": "Колонны пузырей",
+  "blocky-bubbles.options.quality.tooltip": "Влияет на качество колонн пузырей.",
+  "resourcePack.blocky-bubbles:32x_upscale.description": "Основано на §8https://faithfulpack.net",
+  "resourcePack.blocky-bubbles:32x_upscale.name": "Blocky bubbles 32×32"
```

can this line (12, in case github doesn't make it clear!) be rolled back? i would like it to read `Blocky Bubbles: 32x Upscale` just as it is in the english localisation

---

other than that, i'm happy to merge

---

also i will be adding you both to a list of contributors which you can see if you use mod menu :)

---

Oh, sorry, that's my fault
```suggestion
  "resourcePack.blocky-bubbles:32x_upscale.name": "Blocky Bubbles: 32x Upscale"
```